src/equations/numerical_fluxes.jl

```diff
@@ -234,6 +234,36 @@ left and right states `u_ll, u_rr`, usually based only on the local wave speeds
 """
 function min_max_speed_naive end
 
+"""
+    min_max_speed(u_ll, u_rr, orientation::Integer,   equations)
+    min_max_speed(u_ll, u_rr, normal_direction::AbstractVector, equations)
+
+Simple and fast bounds of the minimal and maximal wave speed of the Riemann problem with
+left and right states `u_ll, u_rr`, usually based only on the local wave speeds associated to
+`u_ll` and `u_rr`.
+- Amiram Harten, Peter D. Lax, Bram van Leer (1983)
+  On Upstream Differencing and Godunov-Type Schemes for Hyperbolic Conservation Laws
+  [DOI: 10.1137/1025002](https://doi.org/10.1137/1025002)
+"""
+function min_max_speed end
```

Comment:
If you have this docstring once, it will probably be quite confusing if you add it to every single method, too. For example, I expect that the output of `?min_max_speed` (in help mode) will be quite overwhelming...